src/client/app/app.react.js

```diff
@@ -94,7 +96,7 @@ class App extends Component {
         <RouteHandler {...this.state} />
         <footer>
           <p>
-            made by <a href="https://twitter.com/steida">steida</a>
+            <FormattedHTMLMessage message={msg('app.madeByHtml')} />
```

Comment:
I have yet to figure out a good solution for this. Having the entire tag in the dictionary is awful, but having it as two messages glued together is equaly awful.